flood_adapt/object_model/hazard/hazard.py

```diff
@@ -170,30 +169,30 @@ def run_sfincs(self, site: ISite):
 
         # Run new model (create batch file and run it)
         # create batch file to run SFINCS, adjust relative path to SFINCS executable for ensemble run (additional folder depth)
-        if self.event.attrs.mode == "risk":
-            with open(run_folder_overland.joinpath("run.bat"), "w") as f_out:
-                bat_file: str = (
-                    "cd "
-                    "%~dp0"
-                    "\n"
-                    "..\..\..\..\..\..\..\system\sfincs\{}\sfincs.exe -> sfincs.log".format(
-                        site.attrs.sfincs.version
-                    )
-                )
-                f_out.write(bat_file)
-        elif self.event.attrs.mode == "single_scenario":
-            with open(run_folder_overland.joinpath("run.bat"), "w") as f_out:
-                bat_file: str = (
-                    "cd "
-                    "%~dp0"
-                    "\n"
-                    "..\..\..\..\..\..\system\sfincs\{}\sfincs.exe -> sfincs.log".format(
-                        site.attrs.sfincs.version
-                    )
-                )
-                f_out.write(bat_file)
-
-        subprocess.run(run_folder_overland.joinpath("run.bat"))
+
+        c_dir: str = os.getcwd()
+        file_dir: str = os.path.dirname(__file__)
+
+        file_path = Path(file_dir)
+
+        os.chdir(run_folder_overland)
+
+        levels_up: int = 3
+
+        sfincs_exec: Path = (
+            file_path.parents[levels_up - 1]
+            / "tests"
+            / "system"
+            / "sfincs"
+            / "sfincs.exe"
+        )
+
+        sfincs_log: str = "sfincs.log"
+
+        with open(sfincs_log, "w") as log_handler:
+            subprocess.run(sfincs_exec, stdout=log_handler)
+
+        os.chdir(c_dir)
```

Comment:
Instead of manually changing dir and then going back, copy this function and use it instead:
https://github.com/Deltares/xmipy/blob/f3954fd888e42d24920adc58543185b52f80a6c7/xmipy/utils.py#L6